src/replica/split/replica_split_manager.h

```diff
@@ -0,0 +1,134 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include "replica/replica.h"
```

Comment:
why didn't you name it `partition_split_manager`?

Reply:
The class name is `replica_split_manager`, showing the function is partition_split, and it is used for replica, just like `replica_bulk_loader`.

Reply:
But I saw some comments write `Partition Split`, this will make someone new confused with it.

Reply:
The suitable class name may be `replica_partition_split_manager`, but it is a little long, so I name it `replica_split_manager`, `split` is the abbreviation of `partition split`. Could you please tell me what is your confusion raised by `partition split`?

Reply:
I think it is better to give it a uniform name. or someone without a relevant knowledge background may feel confused with what's the difference between `partition split` and `replica split`

Reply:
Partition Split is the user-facing word. Inside the implementation, child/parent replica both do the split job, in replica level. So `replica_split_manager` is more accurate.

Futhermore, it would be better if child and parent could be separated to different classes.